features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/Swagger/ApiCreateSwagger.jsx

```diff
@@ -176,7 +176,10 @@ class ApiCreateSwagger extends React.Component {
      */
     handleSubmit = (e) => {
         e.preventDefault();
-        if ((this.state.uploadMethod === 'file' && this.state.files.length === 0) || (this.state.uploadMethod === 'url' && !this.state.swaggerUrl)) {
+        if (
+            (this.state.uploadMethod === 'file' && this.state.files.length === 0) ||
+            (this.state.uploadMethod === 'url' && !this.state.swaggerUrl)
```

Comment:
can't we use object destructuring?

Reply:
Fixed in this commit https://github.com/wso2/carbon-apimgt/pull/6604/commits/ea187649180c67929cb429c532d9820dee2f650f#diff-3b47f705e1bc141329665be8d5ab8475L183